src/utilities/MiscUtilities.ts

```diff
@@ -216,4 +220,18 @@ export namespace MiscUtilities {
         }
         return allSections.find(x => x.uniqueIdentifier === result.values[0])!;
     }
+
+        /**
+     * Returns role name when provided with guild and roleId.
+     * @param {string} roleId The role ID.
+     * @param {Guild} guild The guild.
+     * @returns {string} the name of the role or NOT_FOUND
+     */
+        export function getRoleName(roleId: string, guild: Guild): string {
+            LOGGER.debug(`Obtaining role name from ${roleId}`);
+            const role = guild.roles.cache.find(r => r.id === roleId);
+            const ret = role ? role.name : "NOT_FOUND";
```

Comment:
Might be better to have this return `null` if the role wasn't found. That way, in the *off-chance* that someone decides to make a role called `NOT_FOUND` (I know, it's unlikely, but this is a just in case type thing so we don't need to deal with it), that would break this function.